lib/unittest.bzl

```diff
@@ -308,7 +390,22 @@ def _assert_new_set_equals(env, expected, actual, msg = None):
             full_msg = expectation_msg
         _fail(env, full_msg)
 
+def _expect_failure(env, expected_failure_msg = ""):
```

Comment:
Missing doc string since it is a public method?

Reply:
Done